pom.xml

```diff
@@ -97,6 +97,18 @@
       <version>${jackson.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jdk8</artifactId>
```

Comment:
There are several places where optional datatypes can be used in https://github.com/jenkinsci/bitbucket-branch-source-plugin once we switch over to JDK 8 :) So I would very much like it to be there